stringer.go

```diff
@@ -51,11 +54,19 @@ var (
 	transformMethod = flag.String("transform", "noop", "enum item name transformation method (bno. Default: noop")
 	trimPrefix      = flag.String("trimprefix", "", "transform each item name by removing a prefix. Default: \"\"")
 	lineComment     = flag.Bool("linecomment", true, "use line comment text as printed text when present")
-	protoOnly        = flag.Bool("proto-only", false, "whether only generate enum protos and dont do any go generation")
+	protoOnly       = flag.Bool("proto-only", false, "whether only generate enum protos and dont do any go generation")
 	proto           = flag.Bool("proto", false, "wheter or not to generate protobuffer enum counterpart file")
 	protoOutput     = flag.String("proto-output", "", "proto output file name; default srcdir/<protoPkg>/<type>.proto if one type srcdir/<protoPkg>/enums.proto if more than one")
-	protoPkg        =  flag.String("proto-pkg", "", "proto pkg name (default pb)")
-	protoGoPkg 		= flag.String("proto-go-pkg", "", "if supplied will add as value for go_package option added to proto, i.e.  'option go_package = xxxx' ")
+	protoPkg        = flag.String("proto-pkg", "", "proto pkg name (default pb)")
+	protoGoPkg      = flag.String("proto-go-pkg", "", "if supplied will add as value for go_package option added to proto, i.e.  'option go_package = xxxx' ")
+	graphql         = flag.Bool("graphql", false, "whether or not to generate graphql types")
+	graphqlPkg      = flag.String("graphql-pkg", "", "graphql package name (default pb)")
```

Comment:
default pb?

Reply:
Yah -- that's not new on this branch, though.